### PR TITLE
graphql-cli: init at 2.16.5

### PR DESCRIPTION
**Pull request by Zimmi48**
###### Motivation for this change

graphql-cli is a command-line tool to make GraphQL requests.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/development/node-packages/node-env.nix [hunk 1]
```diff
@@ -56,7 +56,7 @@ let
       ) dependencies);
 
   # Recursively composes the dependencies of a package
-  composePackage = { name, packageName, src, dependencies ? [], ... }:
+  composePackage = { name, packageName, src, dependencies ? [], ... }@args:
```
Comment (Infinisil) [inline pkgs/development/node-packages/node-env.nix]: Why this?

Comment (Zimmi48) [reply pkgs/development/node-packages/node-env.nix]: I only applied the documented way of adding a package. I have no idea why it's so non-compositional and I have no idea why this reappeared.

Comment (Infinisil) [reply pkgs/development/node-packages/node-env.nix]: Oh I see, it's auto-generated. Never mind then

**Comment by Infinisil** [comment]
@GrahamcOfBorg build nodePackages.graphql-cli

**Pull request merged**